packages/gatsby-theme-carbon/src/components/Video/video.scss

```diff
@@ -2,6 +2,11 @@
   margin-bottom: $spacing-08;
 }
 
+// video
+.#{$prefix}--video video {
```

Comment:
Do we need the extra element selector here? Don't think we wanna add specificity.

Reply:
Updated the component to remove the extra container divs